### PR TITLE
pcapng: Add a revision number to the docName

### PR DESCRIPTION
**Pull request by fxlb**

### CODE REVIEW

File: draft-tuexen-opsawg-pcapng.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 <rfc category='info'
      ipr='trust200902'
-     docName="draft-tuexen-opsawg-pcapng-latest">
+     docName="draft-tuexen-opsawg-pcapng-latest-00">
```
Comment (guyharris) [inline draft-tuexen-opsawg-pcapng.xml]: This one, on the other hand, should perhaps be -01, as [there appears to be a -00 draft on the IETF site](https://tools.ietf.org/html/draft-tuexen-opsawg-pcapng-00), although it doesn't show up on datatracker.ietf.org.  @mcr, do you know what the status of that draft is?

**Comment by fxlb** [comment]
Already done.

**Comment by mcr** [comment]
Guy Harris <notifications@github.com> wrote:
    >> @@ -10,7 +10,7 @@

    > <rfc category='info'
    > ipr='trust200902'
    > -     docName="draft-tuexen-opsawg-pcapng-latest">
    > +     docName="draft-tuexen-opsawg-pcapng-latest-00">

It needs to be -latest for the new makefile.
It's a bit daft I think, but that's how that makefile works.
I have simpler Makefiles if the group prefers.

**Comment by guyharris** [comment]
-latest, or would -00 also work?  -latest-00 is probably wrong; -latest upsets the IETF RFC linting tool François mentioned.

**Comment by mcr** [comment]
Guy Harris <notifications@github.com> wrote:
    > -latest, or would -00 also work?  -latest-00 is probably wrong; -latest
    > upsets the IETF RFC linting tool François mentioned.

The linting tool is wrong, and/or isn't to be used with that file.
It has to say latest, so that the rest of the Makefile will work.
"make publish" will produce the publishable file.
If you prefer, we can use a simpler Makefile.

**Pull request closed without merging**